test/sqlsmith/mzcompose.py

```diff
@@ -12,14 +12,57 @@
 import time
 from datetime import datetime
 from threading import Thread
-from typing import Any, Dict, FrozenSet, List, Tuple
+from typing import Any, Dict, FrozenSet, List, Optional, Tuple
+
+from materialize.mzcompose import (
+    Composition,
+    Service,
+    ServiceConfig,
+    WorkflowArgumentParser,
+)
+
+
+class StandaloneMaterialized(Service):
+    def __init__(
+        self,
+        name: str,
+        image: Optional[str] = None,
+        ports: List[str] = [],
+        propagate_crashes: bool = True,
+        restart: Optional[str] = None,
+    ) -> None:
+        command = []
+        if propagate_crashes:
+            command += ["--orchestrator-process-propagate-crashes"]
+
+        config: ServiceConfig = {
+            "command": command,
+            "ports": [6875, 6876, 6877, 6878, 26257],
+            "healthcheck": {
+                "test": ["CMD", "curl", "-f", "localhost:6878/api/readyz"],
+                "interval": "1s",
+                # A fully loaded Materialize can take a long time to start.
+                "start_period": "600s",
+            },
+        }
+
+        if image:
```

Comment:
This is only needed if you intend to run this against a specific git tag, the way  misc/rqg/mzcompose runs it. In this case, you can hard-code mzbuild -> materialized as the default.